### PR TITLE
[UI] Display missing page information for /tasks/index

### LINKED ISSUE #1386: [UI] Remove page information from /tasks/index
STR:

1. Navigate to https://taskcluster-ui.herokuapp.com/tasks/index
2. Click on the "?" icon button in the app bar. An empty drawer will pop out.

Screen Shot 2019-09-24 at 1 33 42 PM

We should remove the drawer from this view only.

### PR DESCRIPTION
**Pull request by Rolikasi**
Remove HelpView from Dashboard in tasks/index
I have a lot of commits because I tried to solve the problem when I had very big difference between my PR and original repo. 
Now I understand how branches and remote upstreams work!)))
Closes #1386

### CODE REVIEW

**Comment by Rolikasi** [comment]
solved 1 issue
Now there is no page information button

**Comment by djmitche** [comment]
Thanks!  @helfi92 will be back next week, so let's wait for his review.

**Comment by Rolikasi** [comment]
Ok! Should I continue working on next issues?

**Comment by djmitche** [comment]
Yes, please look at issues that aren't labeled "outreachy".  Thanks!

**Comment by Rolikasi** [comment]
Thank you! I'll take something not labeled as "outreachy"
Also I can help other applicants to work on issues!

**Comment by djmitche** [comment]
That would be great, thanks -- it's very much in the spirit of open source!

**Comment by Rolikasi** [comment]
@helfi92, thanks! 
Done with indexed tasks!)

File: ui/src/views/Tasks/TaskIndex/IndexedTask/index.jsx [hunk 1]
```diff
@@ -98,14 +97,11 @@ export default class IndexedTask extends Component {
...
         loading: indexedTaskLoading,
       },
-      description,
     } = this.props;
     const loading = latestArtifactsLoading || indexedTaskLoading;
 
     return (
-      <Dashboard
-        title="Index Browser"
-        helpView={<HelpView description={description} />}>
```
Comment (helfi92) [inline ui/src/views/Tasks/TaskIndex/IndexedTask/index.jsx]: OH, I just realized that we did have a description set in https://github.com/taskcluster/taskcluster/blob/67af2aff22519eb0433a0e081efba8d512cb90ed/ui/src/views/Tasks/routes.js#L38-L39

but for some reason, it wasn't reaching the views. Let's make sure that description is shown on both views. I think we will need to modify `TaskIndex/routes.js` to have the description. Apologies for not seeing that earlier.

Comment (Rolikasi) [reply ui/src/views/Tasks/TaskIndex/IndexedTask/index.jsx]: Oh, really. Done!)

**Pull request merged**